### PR TITLE
Removed JobId columns

### PR DESCRIPTION
**Pull request by AleksKSoftware**
Removed JobId columns in the tables: 
ChannelOperationRequest, 
Nodes, 
WalletWithdrawalRequest.

### CODE REVIEW

File: src/Migrations/20240123130535_RemoveJobIdColumn.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                table: "ChannelOperationRequests");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Discriminator",
+                table: "AspNetUsers",
+                type: "character varying(21)",
+                maxLength: 21,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "text");
```
Comment (RodriFS) [inline src/Migrations/20240123130535_RemoveJobIdColumn.cs:32]: @Jossec101 I can't find anything about this change in Microsoft documentation or npsql issues. Should we keep it?

Comment (Jossec101) [reply src/Migrations/20240123130535_RemoveJobIdColumn.cs:32]: Looks like a upgrade on NGPSQL did this :/ Lets keep it

File: src/Data/Models/ChannelOperationRequest.cs [hunk 2]
```diff
@@ -150,11 +150,6 @@ public decimal Amount
         [NotMapped]
         public int NumberOfSignaturesCollected => ChannelOperationRequestPsbts == null ? 0 : ChannelOperationRequestPsbts.Count(x => !x.IsFinalisedPSBT && !x.IsTemplatePSBT && !x.IsInternalWalletPSBT);
 
-        /// <summary>
-        /// This is the JobId provided by Quartz of the job executing this request.
-        /// </summary>
-        public string? JobId { get; set; }
-
```
Comment (Jossec101) [inline src/Data/Models/ChannelOperationRequest.cs:157]: Wait, why do we remove this then? Its actually used, though mainly debug @RodriFS

**Review by Jossec101 — CHANGES_REQUESTED** [review]
Worth reviewing imo

**Review by RodriFS — APPROVED** [review]

**Review by RodriFS — APPROVED** [review]

**Comment by AleksKSoftware** [comment]
> Worth reviewing imo

Resolved the issue, I think.

**Review by Jossec101 — APPROVED** [review]

**Pull request merged**